source/manual/graphql.html.md

```diff
@@ -0,0 +1,26 @@
+---
+owner_slack: "#govuk-developers"
+title: GraphQL
+parent: "/manual.html"
+layout: manual_layout
+---
+
+[GraphQL](https://graphql.org/) is an API query language that is used by GOV.UK to serve some live traffic. Our implementation exists in [Publishing API](https://github.com/alphagov/publishing-api) and is used by multiple frontend applications.
+
+Traffic is allocated randomly to either Content Store or GraphQL, to allow us to compare response rates between the two. The decision is made in the frontend applications based on a level of traffic that has been configured through environment variables for each schema (named `GRAPHQL_RATE_SCHEMA_NAME`).
```

Comment:
suggestion (non-blocking): perhaps:
- "semi-randomly" given the mention of "the decision" in the next sentence?
- "response times"? "response rate" sounds more like the number of responses?

Reply:
Those both sound better.  I'll change them.